src/main/java/bio/terra/service/tabulardata/google/BigQueryPdao.java

```diff
@@ -591,8 +592,8 @@ public void createSnapshotWithProvidedIds(
             if (countValue.getLongValue() != rowIds.size()) {
                 logger.error("Invalid row ids supplied: rowIds=" + rowIds.size() +
                     " count=" + countValue.getLongValue());
-                for (String rowId : rowIds) {
-                    logger.error(" rowIdIn: " + rowId);
+                for (UUID rowId : rowIds) {
+                    logger.error(" rowIdIn: " + rowId.toString());
```

Comment:
You don't need `toString()` if the expression is use with a `String` `+` operator.
```suggestion
                    logger.error(" rowIdIn: " + rowId);
```